src/Router/Redirect.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gacela\Router;
+
+final class Redirect
+{
+    public function __construct(
+        private string $uri,
+        private string $destination,
+        private string $method = Request::METHOD_GET,
+        private int $status = 302,
+    ) {
+    }
+
+    public function uri(): string
+    {
+        return $this->uri;
+    }
+
+    public function destination(): string
+    {
+        return $this->destination;
+    }
+
+    public function status(): int
+    {
+        return $this->status;
+    }
+
+    public function method(): string
+    {
+        return $this->method;
+    }
```

Comment:
What about?

```suggestion
    public function __construct(
        public string $uri,
        public string $destination,
        public string $method = Request::METHOD_GET,
        public int $status = 302,
    ) {
    }
```

Let me explain, `property readonly` are available on PHP 8.1 & `class readonly on PHP 8.2.
But in the end, the reason the properties are public/private is to make them not editable from the outside, but as you are the only person working on the project, you won't do that.

Reply:
I like your suggestion, I will do that!